presto-product-tests/src/main/java/io/prestosql/tests/cli/TestPrestoCli.java

```diff
@@ -105,7 +104,7 @@ public void shouldDisplayVersion()
             throws IOException
     {
         launchPrestoCli("--version");
-        String version = firstNonNull(Presto.class.getPackage().getImplementationVersion(), "(version unknown)");
+        String version = firstNonNull(TestPrestoCli.class.getPackage().getImplementationVersion(), "(version unknown)");
```

Comment:
This will make it impossible to run the test from IDE, right?
Maybe we can infer the version from something else, that will be loaded from **a jar** on classpath?
presto jdbc driver maybe, something else? anyone?

Reply:
It's a product test executed by tempto. So it should work